docs/developer-docs/multi-chain/ethereum/using-eth/eth-comparison.mdx

```diff
@@ -0,0 +1,92 @@
+---
+keywords: [intermediate, ethereum, eth, icp]
+---
+```
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Understanding ICP for Ethereum developers
+
+<MarkdownChipRow labels={["Intermediate", "Ethereum"]} />
+````
+
+## Overview
+This guide is designed to help Ethereum developers understand the core concepts of ICP, focusing on the terminology and functionalities that differ between the two protocols. While the underlying concepts might be similar, ICP utilizes different terms to describe them.
+
+For a feature comparison between the two protocols, refer to the [Quick Comparison with ICP and Ethereum](../../../../developer-docs/getting-started/overview-of-icp#quick-comparison-with-ethereum).
+
+## Terminology mapping
+This table summarizes the key terminology differences between Ethereum and ICP:
+
+<table>
+<thead>
+<tr>
+  <th> Ethereum Term </th>
+  <th> ICP Term </th>
+</tr>
+</thead>
+
+<tbody>
+<tr>
+  <td> Smart contract </td>
+  <td> Canister </td>
+</tr>
+
+<tr>
+  <td> Gas </td>
+  <td> Cycles </td>
+</tr>
+
+<tr>
+  <td> Shard </td>
+  <td> Subnet </td>
+</tr>
+
+<tr>
+  <td> Node </td>
+  <td> Replica </td>
+</tr>
+
+<tr>
+  <td> Account </td>
+  <td> Principal </td>
+</tr>
+
+<tr>
+  <td> DAO </td>
+  <td> SNS (ICP has a specific DAO framework named an [SNS](../../../../developer-docs/daos/sns/). However, DAOs can launch without using this framework) </td>
+</tr>
+
+## Smart contract development comparison
+
+Both Ethereum and ICP allow developers to deploy programs that run on the blockchain. These programs are referred to as smart contracts in Ethereum and [canisters](../../../../developer-docs/smart-contracts/overview/) on ICP. Occasionally, the phrase “canister smart contracts” is also used on ICP.
+
+This comparison provides an overview of the key differences in smart contract development on Ethereum and ICP.
+
+## Key differences:
+
+### Creation:
+**Ethereum**: Smart contracts are written in Solidity and deployed immutably onto the blockchain.
+**ICP**: Canisters can be written in any language that complies to WASM including [Rust](../../../../developer-docs/backend/rust/), [Typescript](../../../../developer-docs/backend/Typescript), [Motoko](../../../../developer-docs/backend/motoko/), and [Python](../../../../developer-docs/backend/python/).
+
```

Comment:
We should add a 'Deployment' section that details how Ethereum uses tools like Hardhat/Truffle for deployment while ICP uses dfx.

Reply:
we could also highlight that `scaffold-eth` is similar to our `dfx new` command

Reply:
Thank you for the idea! I will add by EOD tomorrow.